sdk/iot/Azure.Iot.Hub.Service/tests/Azure.Iot.Hub.Service.Tests.csproj

```diff
@@ -15,6 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj" />
```

Comment:
Could you give an idea about what this change does?

Reply:
Sure thing, The azure core team changed every project in master to reference the test project directly but we didn't get that change since we were not in master. This is the new approach we have to take.

Reply:
you can look at this commit as a reference
https://github.com/Azure/azure-sdk-for-net-pr/commit/350cc8d8b9b5eba8330098e98b987fd8cf79277a

Reply:
Cool!